.changeset/modern-weeks-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/countries": patch
+---
+
+Sharp country flags
```

Comment:
This should be an introduction to the change and preferrably contain a code example.

Reply:
Have fleshed out the changeset. Let me know if we need more detail